### PR TITLE
fix: DHIS2-9373 orders coordinates as latitude longitude

### PR DESCRIPTION
**Pull request by paschalidi**
👋 @JoakimSM another fix on the v35 

Ticket: https://jira.dhis2.org/browse/DHIS2-9373

### CODE REVIEW

File: src/core_modules/capture-core/components/Pages/MainPage/WorkingLists/epics/eventsQueryArgsBuilder/filterConverters/optionSet/basicDataTypeConverters.js [hunk 1]
```diff
@@ -32,7 +33,7 @@ const valueConvertersForType = {
     // $FlowFixMe[prop-missing] automated comment
     [elementTypes.IMAGE]: (rawValue: Object) => rawValue.value,
     // $FlowFixMe[prop-missing] automated comment
-    [elementTypes.COORDINATE]: (rawValue: Object) => `[${rawValue.longitude},${rawValue.latitude}]`,
+    [elementTypes.COORDINATE]: MinimalCoordinates,
```
Comment (JoakimSM) [inline src/core_modules/capture-core/components/Pages/MainPage/WorkingLists/epics/eventsQueryArgsBuilder/filterConverters/optionSet/basicDataTypeConverters.js:36]: We should revert this change (this is for converting a coordinate filter value to the api)

**Review by JoakimSM — COMMENTED** [review]
Thanks  @paschalidi. One minor change and we're good to go.

File: src/core_modules/capture-core/components/Pages/MainPage/WorkingLists/epics/eventsQueryArgsBuilder/filterConverters/optionSet/basicDataTypeConverters.js (hunk 1 above)
Comment (paschalidi) [reply src/core_modules/capture-core/components/Pages/MainPage/WorkingLists/epics/eventsQueryArgsBuilder/filterConverters/optionSet/basicDataTypeConverters.js:36]: okay didnt know that. Will take care of it. Thanks 🙏

**Comment by paschalidi** [comment]
@JoakimSM reverted the change as you recommended. Thanks. You might wanna have another look

**Review by JoakimSM — APPROVED** [review]
👍
